modules/overview-of-accelerators.adoc

```diff
@@ -0,0 +1,29 @@
+:_module-type: CONCEPT
+
+[id='overview-of-accelerators_{context}']
+= Overview of accelerators
+
+[role='_abstract']
+If you work with large data sets, you can use accelerators to optimize the performance of your data science models in {productname-short}. With accelerators, you can scale your work, reduce latency, and increase productivity. You can use accelerators in {productname-short} to assist your data scientists in the following tasks:
+
+* Natural language processing (NLP)
+* Inference
+* Training deep neural networks
+* Data cleansing and data processing
+
+{productname-short} supports the following accelerators: 
+
+* NVIDIA graphics processing units (GPUs)
+** To use compute-heavy workloads in your models, you can enable NVIDIA graphics processing units (GPUs) in {productname-short}. 
+**   To enable GPUs on OpenShift, you must install the NVIDIA GPU Operator.
+* Habana Gaudi devices (HPUs)
+** Habana, an Intel company, provides hardware accelerators intended for deep learning workloads. {productname-short} currently supports Habana 1.10.0 and Habana 1.11.0. 
```

Comment:
I'm not sure what we mean by `support` in the upstream, but until there is an appropriate Habana Operator for version 1.11, we shouldn't include it here.

Reply:
Reworded the sentence to remove the word "support". Removed references to 1.11.